components/scream/src/physics/shoc/tests/shoc_varorcovar_tests.cpp

```diff
@@ -270,6 +270,36 @@ struct UnitWrap::UnitTest<D>::TestShocVarorCovar {
     }
   }
 
+
+static void run_bfb()
+  {
+    SHOCVarorcovarData SDS_f90[] = {
+      //               shcol, nlev, nlevi, tunefac
+      SHOCVarorcovarData(10, 71, 72, 1),
+      SHOCVarorcovarData(10, 12, 13, 1),
+      SHOCVarorcovarData(7,  16, 17, 1),
+      SHOCVarorcovarData(2, 7, 8, 0.005),
+    };
+
+    for (auto& d_f90 : SDS_f90) {
```

Comment:
This loop structure diverges from the standard BFB unit test pattern. It is slightly more efficient in terms of lines of code, but there's value in sticking to the expected pattern.